src/components/header/BurgerMenu.vue

```diff
@@ -32,7 +32,9 @@ const route = useRoute();
           <li class="list-group-item text-dark">
             <router-link
               class="text-decoration-none"
-              :class="[route.path === '/' ? 'text-primary' : 'text-secondary']"
+              :class="[
+                route.name === 'dashboard' ? 'text-primary' : 'text-secondary',
```

Comment:
Maybe will be better to move this logic in method?
Arg = route.name

Reply:
ok